dark/titles.py

```diff
@@ -35,6 +35,21 @@ def __init__(self, read, hsps):
         self.read = read
         self.hsps = hsps
 
+    def toDict(self):
+        """
+        Get information about a title alignment as a dictionary.
+
+        @return: A C{dict} representation of the title aligment.
+        """
+        return {
+            'hsps': [hsp.toDict() for hsp in self.hsps],
+            'read': {
+                'id': self.read.id,
+                'quality': self.read.quality,
```

Comment:
This is going to fail if it's not a fastq read, or not?

Reply:
Could you put a toDict method on Read() or something?

Reply:
No, the JSON will just have a `null` if the quality is `None`.

Reply:
Thanks - the Read class already had that method, and now I'm using it :-)